.env

```
@@ -1,2 +1,3 @@
 PORT=8080
+jwtSecret=maxim_bricks_2020
```

Environments are usually in UPPERCASE.
```suggestion
JWT_SECRET=maxim_bricks_2020
```

---

Fixed!